src/test/java/seedu/address/testutil/TypicalEntries.java

```diff
@@ -298,21 +298,23 @@
     public static final String FILE_TEST_CONTENTS = "<!DOCTYPE html>\n<html>\n</html>\n";
 
     // For BrowserPanelTest
+    public static final String BROWSER_PANEL_TEST_ENTRY_BASE_URL =
+            "file://" + MainApp.class.getResource("/view/BrowserPanelTest/test.html")
+                    .toExternalForm().substring(5);
     public static final Entry BROWSER_PANEL_TEST_ENTRY = new EntryBuilder()
             .withTitle("Browser Panel Test Web Page")
             .withDescription("Browser panel test web page")
-            .withLink(MainApp.class.getResource("/view/BrowserPanelTest/test.html").toExternalForm())
+            .withLink(BROWSER_PANEL_TEST_ENTRY_BASE_URL)
             .withAddress("Browser panel test web page")
             .build();
-    public static final String BROWSER_PANEL_TEST_ENTRY_BASE_URL =
-            MainApp.class.getResource("/view/BrowserPanelTest/test.html").toExternalForm();
+    public static final String WIKIPEDIA_ENTRY_BASE_URL = "http://en.wikipedia.org/wiki/Therapsids";
     public static final Entry WIKIPEDIA_ENTRY = new EntryBuilder()
             .withTitle("Wikipedia Test Web Page")
             .withDescription("Wikipedia test web page")
-            .withLink(MainApp.class.getResource("/view/BrowserPanelTest/wikipedia.html").toExternalForm())
+            .withLink("file://" + MainApp.class.getResource("/view/BrowserPanelTest/wikipedia.html")
```

Comment:
Why did we need to revert back to the triple file:/// slash? I thought we patched that particular Link validation regex.

Reply:
It's because of this line in `BrowserPanel#handleSucceeded`:
```java
if (viewMode.getViewType().equals(ViewType.READER)
                && !hasLoadedReaderView
                && currentLocation.equals(webEngine.getLocation())) {
```
I added the last condition in an attempt to fix the bug #104. I'm not exactly sure how the bug happens but this seems to fix it so far.